packages/components/pie-thumbnail/src/index.ts

```diff
@@ -27,21 +29,46 @@ export class PieThumbnail extends LitElement implements ThumbnailProps {
     @property({ type: String })
     public alt = defaultProps.alt;
 
+    @property({ type: Boolean })
+    public disabled = defaultProps.disabled;
+
+    @property({ type: Boolean })
+    public hasPadding = defaultProps.hasPadding;
+
+    @property({ type: String })
+    @validPropertyValues(componentSelector, backgroundColors, defaultProps.backgroundColor)
+    public backgroundColor = defaultProps.backgroundColor;
+
+    @property({ type: Object })
+    public placeholder: ThumbnailProps['placeholder'];
+
+    private handleImageError = () => {
+        if (this.placeholder?.src) this.src = this.placeholder.src as string;
+        if (this.placeholder?.alt) this.alt = this.placeholder.alt as string;
+    };
```

Comment:
the placeholder props don't seem to render as expected in aperture apps 👀 
https://pr239-aperture-nuxt-app.pie.design/components/thumbnail
https://pr239-aperture-vanilla-app.pie.design/components/thumbnail.html

might be the case that we need to swap the values using a state to ensure the component updates correctly?